### DISCUSSION TITLE
Ability to disable the flashing

### DISCUSSION THREAD

**Discussion by mariuszste** [manugarg/pagenotes-issues#61] [General]
> Animate the icon and set badge text when there are notes available for a page.

Please add an option to disable this, it's distracting.

**Comment by manugarg** [comment]
@mariuszste Thank you for the feedback! I've uploaded a release (3.3.3) that will add that option. It should get published within a couple of days.

**Comment by manugarg** [comment]
Release 3.3.3 has been approved by Google. Please update the extension to see the option.
